frame/nis/src/lib.rs

```diff
@@ -217,12 +217,10 @@ pub mod pallet {
 			+ FunMutate<Self::AccountId>
 			+ FunBalanced<Self::AccountId>
 			+ FunHoldInspect<Self::AccountId>
```

Comment:
```suggestion
```
Should be possible to remove. Otherwise we have the possibility of two different `Reason` types.

Reply:
Yeah `Inspect` is already required by `FunMutate`, got find!